src/lib/OscilloscopePanel.svelte

```diff
@@ -16,7 +16,7 @@
  }
 
  function getOscillators (voice: Voice) : number[][] {
-     const oscData = Array(resolution).fill(0).map(() => voice.getOscillators());
+     const oscData = Array(resolution).fill(0).map(() => voice.getOscillators()).slice(1);
```

Comment:
first sample was invalid / bad?

Reply:
Yea the envelope takes 1 sample to kick in. I could try to fix that in the envelope generator, instead.